pkgs/development/python-modules/mne/default.nix

```diff
@@ -78,6 +90,15 @@ buildPythonPackage rec {
     "test_plotting_scalebars"
     # tries to write a datetime object to hdf5, which fails:
     "test_hitachi_basic"
+    # flaky
```

Comment:
since we have urls to bug reports upstream, it would make sense to link to them here so that we know when to clean up skips.